src/js/common/schemaform/sip-api.js

```diff
@@ -29,3 +29,65 @@ export function removeFormApi(formId) {
     return Promise.reject(res);
   });
 }
+
+export function saveFormApi(formId, formData, version, returnUrl, savedAt, trackingPrefix) {
+  // Double stringify because of api reasons. Olive Branch issues, methinks.
+  // TODO: Stop double stringifying
+  const body = JSON.stringify({
+    metadata: JSON.stringify({
+      version,
+      returnUrl,
+      savedAt
+    }),
+    formData: JSON.stringify(formData)
+  });
+
+  const userToken = sessionStorage.userToken;
+  if (!userToken) {
+    Raven.captureMessage('vets_sip_missing_token');
+    window.dataLayer.push({
+      event: `${trackingPrefix}sip-form-save-failed`
+    });
+    return Promise.reject(new Error('Missing token'));
+  }
+
+  return fetch(`${environment.API_URL}/v0/in_progress_forms/${formId}`, {
+    method: 'PUT',
+    headers: {
+      'Content-Type': 'application/json',
+      'X-Key-Inflection': 'camel',
```

Comment:
Not directly related to these changes but, do you have any thoughts on whether it's possible to uncomment this where commented out elsewhere (such as in `removeFormApi` above and `fetchInProgressForm` in `src/js/common/schemaform/sip-api.js`) or why this is needed? Should a bug be filed with the [library](https://github.com/vigetlabs/olive_branch), if one isn't already?

Reply:
I need to check in with Ryan and see if we've updated the backend with the newer version of that library.